Policies/Kubernetes/container-no-privilege-escalation/azurepolicy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "displayName": "[Preview] [PSP]: Do not allow container privilege escalation in Kubernetes cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy does not allow containers to use privilege escalation in a Kubernetes cluster. For instructions on using this policy, please go to https://aka.ms/kubepolicydoc.",
```

Comment:
Can you use below text? David suggests to use this.
For instructions on using this policy, please visit